source/settings/plugin/aclPlugins.rst

```diff
@@ -4,6 +4,25 @@
 acl plugin
 ===============
 
+.. _plugin-acl-setting-acl_ignore_namespace:
+
+``acl_ignore_namespace``
+------------------------
+
+This parameter is used to igore ACLs entirely for the listed namespaces.
+
+.. versionadded:: 2.3.14
+
+Example Setting:
+
+.. code-block:: none
+
+    plugin
+    {
+      acl_ignore_namespace = virtual/
+      acl_ignore_namespace2 = shared/*
```

Comment:
Could have a comment above `# Ignore shared/ and all its (autocreated) child namespaces`